Add README file and simplify Makefile (#3)
rancher-sandbox/cos-toolkit-sample-repo: README.md

# SampleOS

> SampleOS is a dummy example of a derivate system based on containerOS (**cOS**).

Reviewer: ```suggestion
SampleOS is a dummy example of a derivate system built with containerOS-toolkit (**cOS-toolkit**).
```

extend and customize containerOS using its own toolkit.

cOS is built from containers, and completely hosted on image registries. The
build process results in a single container image used to deliver regular
upgrades in OTA approach. Refer to [**cOS project**](https://github.com/rancher-sandbox/cOS-toolkit) for further details
regarding the toolkit or the base cOS system.

This SampleOS only includes a dummy http service on top of the cOS base that
listens to `http://<IP>:8090/fortuneteller` which is enabled through a
cloud-init configuration ([10_sampleOSService.yaml](https://github.com/rancher-sandbox/cos-toolkit-sample-repo/blob/master/packages/sampleOSService/10_sampleOSService.yaml)) applied at
boot time the over the ephemeral `/etc`.

## Quick start

Download the ISO (see actions [artifacts](https://github.com/rancher-sandbox/cos-toolkit-sample-repo/actions)) and boot it with the hipervisor of
your choice. The ISO can be simply boot using KVM with the following commands:

```bash
# create a virtual disk of 20G
$ qemu-img create -f raw test.img 20G

# Run a VM with the ISO
$ qemu-kvm -m 2048 -cdrom sampleOS-0.20210415.iso -hda test.img
```

> After loging into the VM you can check SampleService by typing:

Reviewer: Maybe we should mention default username and password here

```bash
# Service status
systemctl status sampleservice

# Check is up and running properly
curl -L http://localhost:8090/fortuneteller
```

## Build SampleOS Locally

> SampleOS can be build locally by using the following make targets.

Reviewer: Can we also port here the .envrc from cOS? it might be even easier for newcomers
